R/unnest-wider.R

```diff
@@ -0,0 +1,238 @@
+#' Unnest a list-column into columns
+#'
+#' @description
+#' `unnest_wider()` turns each element of a list-column into a column. It
+#' is most naturally suited to list-columns where every element is named,
+#' and the names are consistent from row-to-row.
+#' `unnest_wider()` preserves the rows of `x` while modifying the columns.
+#'
+#' Learn more in `vignette("rectangle")`.
+#'
+#' @export
+#' @param names_sep If `NULL`, the default, the names will be left
+#'   as is. If a string, the outer and inner names will be pasted together using
+#'   `names_sep` as a separator.
+#'
+#'   If the values being unnested are unnamed and `names_sep` is supplied, the
+#'   inner names will be automatically generated as an increasing sequence of
+#'   integers.
+#' @param strict A single logical specifying whether or not to apply strict
+#'   vctrs typing rules. If `FALSE`, typed empty values (like `list()` or
+#'   `integer()`) nested within list-columns will be treated like `NULL` and
+#'   will not contribute to the type of the unnested column. This is useful
+#'   when working with JSON, where empty values tend to lose their type
+#'   information and show up as `list()`.
+#' @family rectangling
+#' @inheritParams unnest_longer
```

Comment:
Same comments about the tag ordering, in this case for both `@inheritParams` and `@export`.

I'm not sure if you really have any strong feelings about where these tags are generally placed, but in my head I have some rough expected order of `inheritParams -> params -> export -> examples`

Reply:
As you can tell, I just stick them anywhere 😁